compiler/ballerina-lang/src/test/java/org/ballerinalang/toml/InvalidSettingsTomlTest.java

```diff
@@ -0,0 +1,39 @@
+/*
+ *  Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package org.ballerinalang.toml;
+
+import org.ballerinalang.compiler.BLangCompilerException;
+import org.ballerinalang.toml.parser.SettingsProcessor;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.nio.file.Paths;
+import java.util.Objects;
+
+public class InvalidSettingsTomlTest {
+
+    @Test(description = "Test invalid Settings.toml", expectedExceptions = BLangCompilerException.class,
+            expectedExceptionsMessageRegExp = ".*invalid Settings.toml due to.*")
```

Comment:
Shall we add the exact due to reason for test as well

Reply:
Added the exact reason